react/src/components/ResourceGroupSelect.tsx

```diff
@@ -27,18 +27,12 @@ const ResourceGroupSelect: React.FC<ResourceGroupSelectProps> = ({
   const currentProject = useCurrentProjectValue();
   const [key, checkUpdate] = useUpdatableState('first');
   const [controllableSearchValue, setControllableSearchValue] =
-    useControllableValue<string>(
-      _.omitBy(
-        {
-          value: searchValue,
-          onChange: onSearch,
-        },
-        _.isUndefined,
-      ),
-    );
-
+    useControllableState<string>({
+      value: searchValue,
+      onChange: onSearch,
```

Comment:
Doesn't it need `_.omitBy`?

Reply:
we wanted to treat the component as uncontrolled component when value was set to undefined,
but useControllableValue treated component as controlled component  if the value property existed in the props,  so we used _.omitBy to not assign the value property when value was undefined.

but now, No need for _.omitBy because I replaced useControllableValue with useControllableState  ( useControllableState treats component as uncontrolled component when value is set to undefined  )